package.json

```diff
@@ -56,14 +56,15 @@
 		"compile": "tsc -p ./",
 		"watch": "tsc -watch -p ./",
 		"pretest": "npm run compile",
-		"test": "nyc mocha -p tsconfig.json --recursive --no-parallel tests/**/*.spec.ts",
+		"test": "nyc mocha -p tsconfig.json --config ./.mocharc.yaml --no-parallel",
```

Comment:
I think the mocharc file should be automatically loaded if it is in the same folder